install/tools/setupIstioVM.sh

```diff
@@ -56,6 +56,7 @@ function istioNetworkInit() {
 # will generate and copy the files and install the packages on each machine.
 function istioInstall() {
   echo "*** Fetching istio packages..."
+  echo "Downloanding istio-sidecar.deb from {PILOT_DEBIAN_URL}/istio-sidecar.deb"
   # Current URL for the debian files artifacts. Will be replaced by a proper apt repo.
   rm -f istio-sidecar.deb
   echo curl -f -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > ${ISTIO_STAGING}/istio-sidecar.deb
```

Comment:
the bug (from #2820) is here

please add the missing "" in that echo and you don't need the first one

Reply:
Done